fireplace/cards/brawl/gift_exchange.py

```diff
@@ -19,8 +19,9 @@ class TB_GiftExchange_Treasure:
 class TB_GiftExchange_Treasure_Spell:
 	# Surely none of this even sort of works.
 	RandomGift = RandomCollectible(
-		COST >= 5,
-		CLASS_CARD=Attr(Controller(OWNER), GameTag.CLASS)
+		card_class=Attr(Controller(OWNER), GameTag.CLASS)
+#		COST >= 5,
+#		CLASS_CARD=Attr(Controller(OWNER), GameTag.CLASS)
```

Comment:
When I commit this revert it will break virtually all of the Travis CI tests just so you know